src/empty-states/__tests__/EmptyState.test.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { LockIcon } from '../../icons'
+import { defaultTheme } from '../../themes'
+import SmallExample from '../fixtures/SmallExample'
+import SmallMinimalExample from '../fixtures/SmallMinimalExample'
+import EmptyState from '../src/EmptyState'
+
+describe('Empty States', () => {
+  it('does not crash when rendering', () => {
+    expect(() =>
+      render(
+        <EmptyState
+          title="My Empty States"
+          image={<LockIcon color={defaultTheme.tokens.colors.gray500} />}
+          imageBgColor={defaultTheme.tokens.colors.gray200}
+        />
+      )
+    ).not.toThrowError()
+  })
+
+  it('should render an image and a title', () => {
+    const { getByText } = render(
+      <EmptyState
+        title="My Empty States"
+        image={<LockIcon color={defaultTheme.tokens.colors.gray500} />}
+        imageBgColor={defaultTheme.tokens.colors.gray200}
+      />
+    )
+    expect(getByText('My Empty States')).toBeInTheDocument()
```

Comment:
What do you think about updating assertions to use `toBeVisible()`? It also includes `toBeInTheDocument()` as implied.